modules/administration-guide/partials/proc_configuring-che-with-custom-resources-to-use-a-rest-backup-server.adoc

```diff
@@ -0,0 +1,62 @@
+[id="configuring-{prod-id-short}-with-custom-resources-to-use-a-rest-backup-server_{context}"]
+= Configuring {prod-short} with custom resources to use a REST backup server
+
+To configure {prod-short} to use a REST backup server:
+
+.Prerequisites
+
+* Configured REST backup server. See xref:supported-restic-compatible-backup-servers.adoc[].
+
+.Procedure
+
+. Create a secret containing the `repo-password` key with a Base64 encoded password:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backup-encryption-password-secret
+  namespace: eclipse-che
+type: Opaque
+stringData:
+  repo-password: __<base64_encoded_password>__
+----
++
+(The {prod-short} Operator setup up this password for the backup repository that the {prod-short} Operator will create from this custom object on the backup server.)
++
+include::snip_warning-about-losing-backup-repo-password.adoc[]
+
+. Create a secret (for example, `name: rest-server-auth-secret`) containing the `username` and `password` keys for the REST server user credentials, using Base64 encoded username and password values.
```

Comment:
`using Base64 encoded username and password values` might confuse. Actually it depends on the way one creates the secret. If it is done via UI or `kubectl create secret ...` no encryption needed (will be done automatically, but if you say to pass already encrypted values we'll end up with double encryption which is wrong. But if the secret is created from yaml via `kubectl apply ...` then base64 encryption needed.

Reply:
@mmorhun What about this?
```suggestion
. Create a secret (for example, `name: rest-server-auth-secret`) containing the `username` and `password` keys for the REST server user credentials.
+
TIP: You don't need to encode the values to Base64 if you create the secret in a UI or by running `kubectl create secret`. However, you must encode the values to Base64 if you create the secret from a YAML file by running `kubectl apply`.
```

Reply:
Also, if the suggested TIP is okay, then should I also add it to
![image](https://user-images.githubusercontent.com/78564304/147232184-d8fd2495-8ab4-401d-8179-3595b93c015d.png)
and to
![Uploading image.png…]()
?

Reply:
That's possible, but this information is more about Kubernetes itself than Che. We can just ask user to create secret with required fields (the same way as any secret is created).